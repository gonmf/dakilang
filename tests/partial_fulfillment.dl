casal("leonardo", Y).
casal(X, "carlota").

casal(X, Y)?
