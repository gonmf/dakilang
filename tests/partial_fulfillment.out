> casal("leonardo", Y).
> casal(X, "carlota").
>
> casal(X, Y)?
X = 'leonardo'
Y = 'carlota'

